charm/ksql/reactive/ksql.py

```diff
@@ -105,6 +112,15 @@ def configure_ksql(kafka):
     ksql_version = ksql.version() or 'unknown'
     hookenv.application_version_set(ksql_version)
 
+@when('website.available', 'ksql.started')
+def setup_website(website):
+    ksql = Ksql()
+
+    # TODO: This currently will not work when there are multiple nodes in the same cluster,
```

Comment:
It should still work for multiple nodes.. it's just that you'd have to configure haproxy with a configuration template in order to get it to work.